tests/modules/nf-core/fastqscreen/fastqscreen/test.yml

```diff
@@ -0,0 +1,10 @@
+- name: "fastqscreen fastqscreen"
+  command: nextflow run ./tests/modules/nf-core/fastqscreen/fastqscreen -entry test_fastqscreen_fastqscreen -c ./tests/config/nextflow.config -c ./tests/modules/nf-core/fastqscreen/fastqscreen/nextflow.config
+  tags:
+    - "fastqscreen"
+    - "fastqscreen/fastqscreen"
+  files:
+    - path: "output/fastqscreen/test_fq_screen/test_1_screen.txt"
```

Comment:
Conda is failing the md5sum, something is different there. Can we test for a string that is contained in the file?